.gitmodules

```
@@ -1,7 +1,7 @@
 [submodule "external/libgit2"]
 	path = external/libgit2
-	url = git://github.com/mono/libgit2.git
+	url = https://www.github.com/mono/libgit2.git
```

This will make it use the same protocol as the parent repository was cloned with, so I think that's a better alternative here.

```suggestion
	url = ../mono/libgit2.git
```